File: servidor/src/main/java/org/example/serverautoescuela/domain/Clase.java
package org.example.serverautoescuela.domain;

import lombok.*;

import java.time.LocalDate;

@Builder
@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
@ToString
public class Clase {
    private int id;
    private LocalDate fecha;
    private String horarioInicio;
    private String duracion;
    private String dniProfesor;
    private String dniAlumno;
}

File: appmovil/app/src/main/java/com/example/autoescuelapp/domain/Clase.kt
package com.example.autoescuelapp.domain

import java.time.LocalDate


data class Clase(
    val fecha: LocalDate?,
    val horarioInicio: String,
    val duracion: String,
    val dniProfesor: String,
    val dniAlumno: String,
    val id: Int? = 0,
)
